### PR TITLE
MAINT: two small fixes for implicit scalar-array-conversions

### PR DESCRIPTION
**Pull request by nschloe**
Last bit of #13864. Two small clean-ups.

### CODE REVIEW

**Comment by nschloe** [comment]
How can I retrigger the build? (I'm assuming the test failure was a blooper.)

**Comment by tylerjereddy** [comment]
Safe to ignore the Azure Windows timeouts for now I think, we've been seeing them a fair bit.

**Comment by nschloe** [comment]
Let me know if there's anything I can do to improve this PR.

**Comment by tupui** [comment]
@nschloe there is something wrong with your PR. The git history seems off. Can you rebase your commits onto master to fix this?

**Comment by nschloe** [comment]
Yeah I just noticed. I'm on it.

**Comment by nschloe** [comment]
Fixed now. Would be happy about a review here! :smiley:

File: scipy/stats/_continuous_distns.py [hunk 1]
```diff
@@ -6655,7 +6655,7 @@ def _preprocess(self, x, skew):
         # close to this small.
         norm2pearson_transition = 0.000016
 
-        ans, x, skew = np.broadcast_arrays([1.0], x, skew)
+        ans, x, skew = np.broadcast_arrays(1.0, x, skew)
```
Comment (tirthasheshpatel) [inline scipy/stats/_continuous_distns.py:6658]: What does this have to do with scalar conversion? IIUC, none of the three variables are being coerced from array to float, right?

**Review by tirthasheshpatel — COMMENTED** [review]
Looked at #13864 and both the changes here seem in line with it. I just have one question.

File: scipy/stats/_continuous_distns.py (hunk 1 above)
Comment (nschloe) [reply scipy/stats/_continuous_distns.py:6658]: Not at this point, but using `[1.0]` instead of `1.0` adds an unnecessary dimension ifi `x` and `skew` are scalars. This then leads to issues down the line.

Comment (tirthasheshpatel) [reply scipy/stats/_continuous_distns.py:6658]: Ah, OK. Makes sense.

**Review by tirthasheshpatel — APPROVED** [review]
I don't have any other comments. Pretty straightforward fixes. Thanks, @nschloe!

**Comment by tupui** [comment]
Feel free to merge @tirthasheshpatel :)

**Review by tylerjereddy — APPROVED** [review]
100 % of the patch diff is covered by tests and reviewers seem happy, so in it goes

**Comment by tylerjereddy** [comment]
thanks @nschloe et al.
